src/britive/secrets_manager.py

```diff
@@ -370,6 +370,20 @@ def update(self, vault_id: str, path: str = '/', value: dict = {}) -> None:
             f'{self.base_url}/{vault_id}/secrets?path={path}', json={'value': value}
         )
 
+    def rename(self, vault_id: str, path: str = '/', new_name: str = '') -> None:
+        """
+        Updates a secret's value
```

Comment:
change to: Update the name of a secret.

Reply:
changed the comment to "Update the name of a secret"